app/helpers/alchemy/pages_helper.rb

```diff
@@ -64,9 +64,13 @@ def render_page_layout
     #
     def render_site_layout(&block)
       render current_alchemy_site, &block
-    rescue ActionView::MissingTemplate
-      warning("Site layout for #{current_alchemy_site.try(:name)} not found. Please run `rails g alchemy:site_layouts`")
-      ""
+    rescue ActionView::MissingTemplate => error
+      if Rails.env.production?
```

Comment:
What about `staging?`? 🤔 

I think we should maybe use some kind of `Rails.application.config` that is used by Rails for this use case anyway? 

Maybe `config.consider_all_requests_local`? 

According the documentation it could be interpreted as for this case

```rb
# config/environment/development.rb

# Show full error reports.
config.consider_all_requests_local = true
```

```rb
# config/environment/production.rb

# Full error reports are disabled
config.consider_all_requests_local = false
```

A weird name, maybe there is something better?

Reply:
This looks promising

https://guides.rubyonrails.org/configuring.html#config-action-dispatch-show-exceptions

Reply:
`config.action_dispatch.show_exceptions` is only available from Rails 7.1, and we still support 7.0. I went with `config.consider_all_requests_local` instead, which reflects pretty exactly what we want: "Local" requests have a developer looking at them, non-local ones a normal user.

Reply:
I think it is present in Rails 7.0 but the default has changed in 7.1

![image](https://github.com/AlchemyCMS/alchemy_cms/assets/42868/e661267e-b562-4af6-b210-f0b80857dcbb)

Reply:
But I also agree that consider all requests local is what we want in this case.